### ISSUE TITLE
cli problem. Integer overflow when casting to u64

### ISSUE THREAD

**Issue by shiharuharu** [filecoin-saturn/contracts#65]
```
thread 'main' panicked at 'Integer overflow when casting to u64', /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/primitive-types-0.12.1/src/lib.rs:38:1
stack backtrace:
   0: rust_begin_unwind
             at /rustc/dd5d7c729d4e8a59708df64002e09dbcbc4005ba/library/std/src/panicking.rs:593:5
   1: core::panicking::panic_fmt
             at /rustc/dd5d7c729d4e8a59708df64002e09dbcbc4005ba/library/core/src/panicking.rs:67:14
   2: primitive_types::U256::as_u64
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/uint-0.9.5/src/uint.rs:647:6
   3: cli::utils::inspect_earnings::{{closure}}
             at /root/contracts/cli/src/utils.rs:673:22
   4: cli::commands::Cli::run::{{closure}}
             at /root/contracts/cli/src/commands.rs:352:73
   5: saturn_contracts::main::{{closure}}
             at /root/contracts/cli/src/bin/saturn-contracts.rs:10:36
   6: tokio::runtime::park::CachedParkThread::block_on::{{closure}}
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.28.2/src/runtime/park.rs:283:63
   7: tokio::runtime::coop::with_budget
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.28.2/src/runtime/coop.rs:107:5
   8: tokio::runtime::coop::budget
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.28.2/src/runtime/coop.rs:73:5
   9: tokio::runtime::park::CachedParkThread::block_on
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.28.2/src/runtime/park.rs:283:31
  10: tokio::runtime::context::BlockingRegionGuard::block_on
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.28.2/src/runtime/context.rs:315:13
  11: tokio::runtime::scheduler::multi_thread::MultiThread::block_on
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.28.2/src/runtime/scheduler/multi_thread/mod.rs:66:9
  12: tokio::runtime::runtime::Runtime::block_on
             at /root/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.28.2/src/runtime/runtime.rs:304:45
  13: saturn_contracts::main
             at /root/contracts/cli/src/bin/saturn-contracts.rs:9:5
  14: core::ops::function::FnOnce::call_once
             at /rustc/dd5d7c729d4e8a59708df64002e09dbcbc4005ba/library/core/src/ops/function.rs:250:5
```

**Comment by AmeanAsad** [comment]
@shiharuharu Issued a fix in #66, please re-install the CLI and try the inspect command again :)

**Comment by AmeanAsad** [comment]
@shiharuharu Closing as fix was issued.
